tfidf_average_rouge_1_2_l.py

```diff
@@ -0,0 +1,140 @@
+import json
+import pickle
+import os
+import numpy as np
+from rouge import Rouge
+import sys
+
+"""
+Discription:
+Code computes scores of sentences for every text, and extracts top sentences as summary.
+
+Arguments:
+argv[1] - tfidf pickeled model path
+argv[2] - file with test filenames
+argv[3] - absolute path of directory which contains lemmatized texts
+argv[4] - count of sentences to extract as summary
+"""
+
+
+
+def process_title(text):
+    processed_text = ''
+    for i in range(len(text)):
+        processed_text += ' '.join(text[i]) + ' '
+    return processed_text
+
+
+def process_sentence(text):
+    processed_text = ''
+    for i in range(len(text)):
+        processed_text += ''.join(text[i]) + ' '
+    return processed_text
+
+
+def rouge_score(hypothesis, summary):
+    rouge = Rouge()
+    score = rouge.get_scores(hypothesis, summary)[0]
+    return score
+
+
+def print_results(sample):
+    for key in sample:
+        print(key)
+        print("rouge_1 - ", sample[key][0] / sample[key][3])
+        print("rouge_2 - ", sample[key][1] / sample[key][3])
+        print("rouge_l - ", sample[key][2] / sample[key][3])
+
+
+def load_model(model):
+    with open(model, 'rb') as f:
+        tf_idf_model = pickle.load(f)
+    return tf_idf_model
+
+
+def get_test_filenames(file):
+    with open(file, 'r') as f:
+        file_names = f.readlines()
+    return file_names
+
+
+def get_hypothesis(text, scores):
+    hypothesis = []
+    sentence = ''
+    for i in range(0, 3):
+        if len(text) >= i + 1:
+            sentence += process_sentence(text[np.argmax(scores)])
+            hypothesis.append(sentence)
+            scores[np.argmax(scores)] = -1
+        else:
+            break
+    return hypothesis
+
+
+def process_lemma(text):
```

Comment:
The function is called `process_lemma` but it's not doing anything related to lemmas. It looks like it just concatenates parts of each word and then joins them with a whitespace separator.